modules/@apostrophecms/schema/lib/addFieldTypes.js

```diff
@@ -256,7 +256,7 @@ module.exports = (self) => {
   self.addFieldType({
     name: 'color',
     async convert(req, field, data, destination) {
-      destination[field.name] = self.apos.launder.string(data[field.name], field.def);
+      destination[field.name] = self.apos.launder.string(data[field.name], destination[field.name] || undefined);
```

Comment:
In this case, `undefined` is a sensible value. But you shouldn't need `destination[field.name]` here.